### PR TITLE
@ashfurrow -> make sure section dimensions reflect header height even if there are no items

### PR DESCRIPTION
**Pull request by 1aurabrown**
Footer location was getting calculated to be in the wrong place if the longest/shortest sections hadn't yet been calculated.

### CODE REVIEW

**Comment by 1aurabrown** [comment]
@orta @everyone take a look, if nobody has any objections i'm going to merge and tag so that eigen can use this change -- it affects the Favorites view.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+2.0.1
+----
+
+- [#25](https://github.com/ashfurrow/ARCollectionViewMasonryLayout/pull/25): Section dimensions and max attribtue should reflect header height even if items haven't yet been added to the collection view. - [@1aurabrown](https://github.com/1aurabrown)
```
Comment (orta) [inline CHANGELOG.md]: minor typo

**Comment by orta** [comment]
This looks good to me @1aurabrown

**Comment by 1aurabrown** [comment]
i didn't mean to close this

**Comment by 1aurabrown** [comment]
Fixed our changelog too. Somehow 1.0.1 and 2.0.0 became the same thing in the changelog, i splt them up. Some of the releases were also missing so I tried to add them to the changelog.

**Comment by 1aurabrown** [comment]
I'm going to merge

**Pull request merged**